### DISCUSSION TITLE
The Package initializer is deprecated in Package@swift-6.0

### DISCUSSION THREAD

**Discussion by NoFearJoe** [siteline/SwiftUI-Introspect#442] [Ideas]
I get the next warning when fetch the SwiftUI-Introspect dependency:

`init(name:defaultLocalization:platforms:pkgConfig:providers:products:dependencies:targets:swiftLanguageVersions:cLanguageStandard:cxxLanguageStandard:)' is deprecated: replaced by 'init(name:defaultLocalization:platforms:pkgConfig:providers:products:dependencies:targets:swiftLanguageModes:cLanguageStandard:cxxLanguageStandard:)`

Are there any reasons to use the deprecated init? Could you please exchange it for the new one?

**Comment by davdroman** [comment]
Fixed in https://github.com/siteline/swiftui-introspect/pull/444
